kubejs/startup_scripts/script.js

```diff
@@ -50,7 +50,7 @@ onEvent('item.registry', function (event) {
     food.eaten(function (e) {
       if (!e.player.stages.has('nether_access')) {
         e.player.stages.add('nether_access');
-        e.player.tell(text.green("You have unlocked access to The Nether"));
+        e.player.tell(text.green("您已解锁访问下界"));
```

Comment:
https://minecraft.fandom.com/zh/wiki/%E4%B8%8B%E7%95%8C?variant=zh
```suggestion
        e.player.tell(text.green("您已解锁进入下界的权限"));
```